### ISSUE TITLE
Using 'setAttribute' to change the value of a label results in character overlay

### ISSUE THREAD

**Issue by mauriciomartinez-itesm** [rdub80/aframe-gui#25]
This worked with the previous version of aframe-gui and is now broken, here are the results now:

![capture](https://user-images.githubusercontent.com/33909685/47318354-75909d80-d611-11e8-92ce-44ca9fd55abc.PNG)

This happens when you try to change the value like this after a value was already given to the label.

`document.querySelector('#myLabel').setAttribute("value", "New Value");`

```
<a-gui-label
          id="myLabel"
          value="OldValue">
</a-gui-label>
```

**Comment by rdub80** [comment]
what version on A-Frame are you currently using?

**Comment by mauriciomartinez-itesm** [comment]
https://aframe.io/releases/0.8.2/aframe.min.js

0.8.2

**Comment by iburguera** [comment]
Any update regarding this issue? I still get the same error

**Comment by lmalave** [comment]
Sorry for the delay - I will prioritize resolving this issue.

**Comment by lmalave** [comment]
This has been fixed as of version 0.3.3
